frontendExternalSystem/build.gradle

```diff
@@ -0,0 +1,9 @@
+vavr()
+
+addIntellijToCompileClasspath(withGit4Idea: true)
+
+dependencies {
+  implementation project(":logging")
```

Comment:
Nit: use `'` instead of `"` in gradle files unless needed for string interpolation (which is available under `"` but not `'`)